libs/remix-ui/workspace/src/lib/actions/index.ts

```diff
@@ -80,11 +80,13 @@ export const initWorkspace = (filePanelPlugin) => async (reducerDispatch: React.
             {id: 5, name: 'goerli'}
           ]
           let found = false
+          const workspaceName = 'etherscan-code-sample'
+          let filePath
           const foundOnNetworks = []
           for (const network of networks) {
             const target = `/${network.name}/${contractAddress}`
             try {
-              data = await fetchContractFromEtherscan(plugin, network, contractAddress, target, etherscanKey)
+              data = await fetchContractFromEtherscan(plugin, network, contractAddress, target, false, etherscanKey)
```

Comment:
if you `shouldSetFile` to false, you won't write the files to the file explorer right?

Reply:
In this case, it tries to set file before setting/creating workspace  which shows errors in browser console like:
`Error: Error: ENOENT: /.workspaces/etherscan-code-sample/mainnet/0xdac17f958d2ee523a2206206994597c13d831ec7/TetherToken.sol`
We set the file again after workspace creation.